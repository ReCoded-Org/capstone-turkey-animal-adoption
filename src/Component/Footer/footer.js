import React from "react";
import {
  FaFacebookF,
  FaTwitter,
  FaPinterestP,
  FaInstagram,
  FaLinkedin,
} from "react-icons/fa";
import "bootstrap/dist/css/bootstrap.min.css";
import logo from "./logo.png";
import dog from "./Bg.png";

const Footer = () => {
  return (
    <footer className="text-center text-lg-start mt-4 pb-3">
      <div className="container p-2">
        <div className="row">
          <div className="col">
            <div className="d-flex align-items-center pb-4 pt-4">
              <div className="p-2">
                <img src={logo} alt="my logo" width="160" height="120" />
              </div>
              <div className="p-2">
                <h3>Furry</h3>
                <h3>Friends</h3>
              </div>
            </div>
            <div className="pt-4 text-white">
              Copyright Â© All rights reserved for Re:Coded
            </div>
          </div>

          <div className="col">
            <h4>PRODUCTS</h4>
            <ul className="list-group">
              <li className="list-unstyled text-white">Features </li>
              <li className="list-unstyled text-white">Personal License </li>
              <li className="list-unstyled text-white">Personal License </li>
              <li className="list-unstyled text-white">Pricing Options </li>
              <li className="list-unstyled text-white">Personal License</li>
            </ul>
          </div>

          <div className="col pb-2">
            <h4>LATEST POSTS</h4>
            <ul className="list-group pb-2">
              <div className=" d-flex align-items-center">
                <img
                  src={dog}
                  alt="dog"
                  width="80"
                  height="60"
                  className="m-1"
                />
                <li className="list-unstyled text-white ">
                  Lawyaw uses AI to help lawyers draft documents faster
                </li>
              </div>
              <div className=" d-flex align-items-center pt-2">
                <img
                  src={dog}
                  alt="dog"
                  width="80"
                  height="60"
                  className="m-1"
                />
                <li className="list-unstyled text-white">
                  Going against the grain, AngelPad kills its demo day
                </li>
              </div>
            </ul>

            <div className="d-sm-flex justify-content-sm-center pt-2">
              <div className="p-2">
                <FaFacebookF />
              </div>
              <div className="p-2">
                <FaTwitter />
              </div>
              <div className="p-2">
                <FaPinterestP />
              </div>
              <div className="p-2 ">
                <FaInstagram />
              </div>
              <div className="p-2">
                <FaLinkedin />
              </div>
            </div>
          </div>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
