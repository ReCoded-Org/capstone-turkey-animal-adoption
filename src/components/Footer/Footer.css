.page-footer {
  background-color: #31393c;
  color: white;
}
