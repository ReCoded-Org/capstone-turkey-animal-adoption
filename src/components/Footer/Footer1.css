footer {
  background-color: #31393c;
  color: white;
}
.page-footer {
  background-color: #31393c;
  color: white;
}

.headersColor {
  color: #ff7777;
  text-align: left;
}

.ulJustify {
  text-align: left;
}
.footer-copyright {
  text-align: left;
  font-size: 14px;
}
footer {
  background-color: #31393c;
  color: white;
}

.dogImage {
  width: 30;
  height: 20;
  border-radius: 5px;
}
footer h3 {
  color: hsl(0, 100%, 73%);
  text-align: left;
}

footer h4 {
  color: #ff7777;
  text-align: left;
}

.dogImage {
  width: 30;
  height: 20;
  border-radius: 5px;
}

footer li {
  text-align: left;
}
