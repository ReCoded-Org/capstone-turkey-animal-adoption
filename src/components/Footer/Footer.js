import React from "react";
import {
  FaFacebookF,
  FaTwitter,
  FaPinterestP,
  FaInstagram,
  FaLinkedin,
} from "react-icons/fa";
import logo from "./logo.png";
import { NavLink } from "react-router-dom";
import dog from "./Bg.png";
import "./Footer.css";
import { useTranslation } from "react-i18next";

const Footer = () => {

  const { t } = useTranslation();

  return (
    <footer className="text-center text-lg-start mt-5 pb-3">
      <div className="container p-2">
        <div className="row">
          <div className="col-md-4">
            <div className="d-flex align-items-center pb-4 pt-4">
              <div className="p-2">
                <NavLink to="/">
                  <img src={logo} alt="my logo" width="100" />
                </NavLink>
              </div>
              <div className="p-2">
                <h3>Furry</h3>
                <h3>Friends</h3>
              </div>
            </div>
            <div className="pt-4 text-white">
            {t("footer.allRights")}
            </div>
          </div>
          <div className="col-md-4 pl-lg-5 pl-md-5 pl-0 mt-3">
            <h4>{t("footer.title0")}</h4>
            <ul className="list-group">
              <li className="list-unstyled text-white">
                <NavLink to="/" className="text-white">
                {t("footer.subTitleHome")}
                </NavLink>
              </li>
              <li className="list-unstyled text-white">
                <NavLink to="/about" className="text-white">
                {t("footer.subTitleAbout")}
                </NavLink>
              </li>
              <li className="list-unstyled text-white">
                <NavLink to="/search" className="text-white">
                {t("footer.subTitleFindPet")}
                </NavLink>
              </li>
              <li className="list-unstyled text-white">
                <NavLink to="/blog" className="text-white">
                {t("footer.subTitleBlog")}
                </NavLink>
              </li>
              <li className="list-unstyled text-white">
                <NavLink to="/contact" className="text-white">
                {t("footer.subTitleContact")}
                </NavLink>
              </li>
              <li className="list-unstyled text-white">
                <NavLink to="/resources" className="text-white">
                {t("footer.subTitleResources")}
                </NavLink>
              </li>
            </ul>
          </div>
          <div className="col-md-4  mt-3 pl-lg-5 pl-0">
            <h4>{t("footer.title1")}</h4>
            <ul className="list-group pb-2">
              <div className=" d-flex align-items-center">
                <img
                  src={dog}
                  alt="dog"
                  width="80"
                  height="60"
                  className="m-1"
                />
                <li className="list-unstyled text-white ">
                  Lawyaw uses AI to help lawyers draft documents faster
                </li>
              </div>
              <div className="d-flex align-items-center pt-2">
                <img
                  src={dog}
                  alt="dog"
                  width="80"
                  height="60"
                  className="m-1"
                />
                <li className="list-unstyled text-white">
                  Going against the grain, AngelPad kills its demo day
                </li>
              </div>
            </ul>
            <div className="d-sm-flex justify-content-sm-center pt-2">
              <div className="p-2">
                <FaFacebookF />
              </div>
              <div className="p-2">
                <FaTwitter />
              </div>
              <div className="p-2">
                <FaPinterestP />
              </div>
              <div className="p-2 ">
                <FaInstagram />
              </div>
              <div className="p-2">
                <FaLinkedin />
              </div>
            </div>
          </div>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
