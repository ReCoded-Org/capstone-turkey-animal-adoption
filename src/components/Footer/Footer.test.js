import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import Footer from "./Footer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Router>
        <Footer />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
