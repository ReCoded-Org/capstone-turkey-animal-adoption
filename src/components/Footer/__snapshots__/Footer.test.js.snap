// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="text-center text-lg-start mt-4 pb-3"
>
  <div
    className="container p-2"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <div
          className="d-flex align-items-center pb-4 pt-4"
        >
          <div
            className="p-2"
          >
            <a
              aria-current="page"
              className="active"
              href="/"
              onClick={[Function]}
              style={Object {}}
            >
              <img
                alt="my logo"
                src="logo.png"
                width="100"
              />
            </a>
          </div>
          <div
            className="p-2"
          >
            <h3>
              Furry
            </h3>
            <h3>
              Friends
            </h3>
          </div>
        </div>
        <div
          className="pt-4 text-white"
        >
          Copyright Â© All rights reserved for Re:Coded
        </div>
      </div>
      <div
        className="col pl-5 mt-2"
      >
        <h4
          className="pl-5"
        >
          Menu
        </h4>
        <ul
          className="list-group pl-5"
        >
          <li
            className="list-unstyled text-white"
          >
            <a
              aria-current="page"
              className="text-white active"
              href="/"
              onClick={[Function]}
              style={Object {}}
            >
              Home
            </a>
          </li>
          <li
            className="list-unstyled text-white"
          >
            <a
              aria-current={null}
              className="text-white"
              href="/about"
              onClick={[Function]}
            >
              About
            </a>
          </li>
          <li
            className="list-unstyled text-white"
          >
            <a
              aria-current={null}
              className="text-white"
              href="/search"
              onClick={[Function]}
            >
              Find A Pet
            </a>
          </li>
          <li
            className="list-unstyled text-white"
          >
            <a
              aria-current={null}
              className="text-white"
              href="/blog"
              onClick={[Function]}
            >
              Blog
            </a>
          </li>
          <li
            className="list-unstyled text-white"
          >
            <a
              aria-current={null}
              className="text-white"
              href="/contact"
              onClick={[Function]}
            >
              Contact Us
            </a>
          </li>
          <li
            className="list-unstyled text-white"
          >
            <a
              aria-current={null}
              className="text-white"
              href="/resources"
              onClick={[Function]}
            >
              Resources
            </a>
          </li>
        </ul>
      </div>
      <div
        className="col  mt-2"
      >
        <h4>
          LATEST POSTS
        </h4>
        <ul
          className="list-group pb-2"
        >
          <div
            className=" d-flex align-items-center"
          >
            <img
              alt="dog"
              className="m-1"
              height="60"
              src="Bg.png"
              width="80"
            />
            <li
              className="list-unstyled text-white "
            >
              Lawyaw uses AI to help lawyers draft documents faster
            </li>
          </div>
          <div
            className="d-flex align-items-center pt-2"
          >
            <img
              alt="dog"
              className="m-1"
              height="60"
              src="Bg.png"
              width="80"
            />
            <li
              className="list-unstyled text-white"
            >
              Going against the grain, AngelPad kills its demo day
            </li>
          </div>
        </ul>
        <div
          className="d-sm-flex justify-content-sm-center pt-2"
        >
          <div
            className="p-2"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 320 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
              />
            </svg>
          </div>
          <div
            className="p-2"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
          </div>
          <div
            className="p-2"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 384 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M204 6.5C101.4 6.5 0 74.9 0 185.6 0 256 39.6 296 63.6 296c9.9 0 15.6-27.6 15.6-35.4 0-9.3-23.7-29.1-23.7-67.8 0-80.4 61.2-137.4 140.4-137.4 68.1 0 118.5 38.7 118.5 109.8 0 53.1-21.3 152.7-90.3 152.7-24.9 0-46.2-18-46.2-43.8 0-37.8 26.4-74.4 26.4-113.4 0-66.2-93.9-54.2-93.9 25.8 0 16.8 2.1 35.4 9.6 50.7-13.8 59.4-42 147.9-42 209.1 0 18.9 2.7 37.5 4.5 56.4 3.4 3.8 1.7 3.4 6.9 1.5 50.4-69 48.6-82.5 71.4-172.8 12.3 23.4 44.1 36 69.3 36 106.2 0 153.9-103.5 153.9-196.8C384 71.3 298.2 6.5 204 6.5z"
              />
            </svg>
          </div>
          <div
            className="p-2 "
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              />
            </svg>
          </div>
          <div
            className="p-2"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
